android/gradle.properties

```diff
@@ -17,3 +17,10 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemor
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 RNGH_kotlinVersion=1.6.21
+
+# Path to codegen output directory with this library view managers' interfaces & delegates. Used by `copyCodegenArtifacts` task that helps to synchronize newly generated files with their Paper counterparts.
+codegenArtifactsSource=android/build/generated/source/codegen/java/com/facebook/react/viewmanagers
+
+# Path to directory with view managers' interfaces & delegates used while running on Paper architecture. This property is used as the output path for `copyCodegenArtifacts` task.
+# Used for task (copyCodegenArtifacts) that automates copying those interfaces/delegates when codegen is run
```

Comment:
```suggestion
# Used by `copyCodegenArtifacts` task that automates copying these interfaces/delegates while codegen is running.
```

Reply:
`while codegen is running.` if anything, imo it should be `after codegen is run`

Reply:
Changed in: https://github.com/software-mansion/react-native-gesture-handler/pull/2933/commits/7e4da8d6e5dcc10337a96a60d5e4c44c6d75283b